.github/workflows/next-build-multiarch.yml

```diff
@@ -106,5 +104,5 @@ jobs:
             echo "[!] The job 'build-images' didn't provide any outputs. Can't create the manifest list."
             exit 1;
           fi
-          docker manifest create ${{ env.IMAGE }}:multiarch-next $AMEND
-          docker manifest push ${{ env.IMAGE }}:multiarch-next
+          docker manifest create ${{ env.IMAGE }}:next $AMEND
```

Comment:
Well, it also requires adapting PR check not to pass when multiarch build fails.
And it has a drawback... For some reason amd64 build takes 4-6 minutes, while other 35-45 minutes which is not critical but not pleasure when you add a rewording fix in the PR before merging.